packages/smarthr-ui/src/components/ResponseMessage/stories/ResponseMessage.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Stack } from '../../Layout'
+
+import { ResponseMessage, classNameGenerator } from '../ResponseMessage'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+export default {
+  title: 'Text（テキスト）/ResponseMessage',
+  component: ResponseMessage,
+  render: (args) => <ResponseMessage {...args} />,
+  args: {
+    children: 'レスポンスメッセージ',
+  },
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+} satisfies Meta<typeof ResponseMessage>
+
+export const Playground: StoryObj<typeof ResponseMessage> = {
+  args: {},
+}
+
+export const Type: StoryObj<typeof ResponseMessage> = {
+  name: 'type',
+  render: (args) => (
+    <Stack align="flex-start">
+      {[undefined, ...Object.keys(classNameGenerator.variants.type)].map((type) => (
+        <ResponseMessage {...args} type={type as any} key={type} />
+      ))}
+    </Stack>
+  ),
+}
+
+export const Size: StoryObj<typeof ResponseMessage> = {
+  name: 'size',
+  render: (args) => (
+    <Stack align="flex-start">
+      <ResponseMessage {...args} size="XXS" />
+      <ResponseMessage {...args} size="XS" />
+      <ResponseMessage {...args} size="S" />
+      <ResponseMessage {...args} size="M" />
+      <ResponseMessage {...args} size="L" />
+      <ResponseMessage {...args} size="XL" />
+      <ResponseMessage {...args} size="XXL" />
+    </Stack>
+  ),
+}
+
+export const Right: StoryObj<typeof ResponseMessage> = {
+  name: 'right',
+  render: (args) => (
+    <Stack align="flex-start">
+      <ResponseMessage {...args} />
+      <ResponseMessage {...args} right />
+    </Stack>
```

Comment:
right の Story なので right だけでよさそう。

```suggestion
    <ResponseMessage {...args} right />
```

Reply:
修正済